### PR TITLE
Ldc fix

### PR DESCRIPTION
**Pull request by hover2pi**
I added multiprocessing to `core.ModelGrid()` for grid interpolation, added errors and multiple-profile input to `ldc.ldcfit.ldc()`, and removed the `ldc.ldcfit` import in `ldc.ldcplot`.

### CODE REVIEW

File: ExoCTK/ldc/ldcfit.py [hunk 1]
```diff
@@ -10,7 +10,6 @@
 import matplotlib.pyplot as plt
 from matplotlib import rc
 from scipy.optimize import curve_fit
-from scipy.interpolate import RegularGridInterpolator
 try:
```
Comment (mattjhill) [inline ExoCTK/ldc/ldcfit.py]: Remove the try/except statements. We want to always use relative imports within the package.  `from ExoCTK import` only works if you have a previous version installed and doesn't solve the issue.

File: ExoCTK/ldc/ldcplot.py [hunk 2]
```diff
@@ -9,182 +9,226 @@
 import os
 import copy
 import inspect
-from bokeh.plotting import figure
+from bokeh.plotting import figure, show
+from bokeh.models import Span
 from matplotlib import rc, cm
 from astropy.io import fits
```
Comment (mattjhill) [inline ExoCTK/ldc/ldcplot.py]: Again remove try/except staments

File: ExoCTK/ldc/ldcfit.py (hunk 1 above)
Comment (hover2pi) [inline ExoCTK/ldc/ldcfit.py]: Done.

File: ExoCTK/ldc/ldcplot.py (hunk 2 above)
Comment (hover2pi) [inline ExoCTK/ldc/ldcplot.py]: Done.

**Pull request merged**